pkg/cybr/conjur/logon.go

```diff
@@ -44,7 +44,7 @@ func Login(applianceURL string, account string, username string, password []byte
 		return nil, err
 	}
 
-	url := fmt.Sprintf("%s/authn/%s/login", applianceURL, url.QueryEscape(account))
+	url := fmt.Sprintf("%s/%s/login", applianceURL, url.QueryEscape(account))
```

Comment:
`applianceURL` should be changed to `authnURL`

Reply:
Is that just for readability?  (Just wondering...)

Reply:
Yes, authnURL shows that it is a `authnURL` and it is not a `applianceURL`